Add `conditionNames: ['import']` to the default webpack config?

I ran into a library that I couldn't import via ember-auto-import without additional webpack config.

https://www.npmjs.com/package/@googleworkspace/drive-picker-element

This package publishes an ESM import path like this:
```js
import {
  DrivePickerElement,
  DrivePickerDocsViewElement,
} from "@googleworkspace/drive-picker-element/drive-picker";
```

ember-auto-import can't resolve this line by default, emitting a webpack error:
```
Module not found: Error: Package path ./drive-picker is not exported from package
```
The package.json exports list for this package is as follows
```json
"exports": {
	".": {
		"browser": "./dist/index.iife.min.js",
		"types": "./dist/index.d.ts",
		"import": "./dist/index.js"
	},
	"./drive-picker": {
		"types": "./dist/drive-picker/index.d.ts",
		"import": "./dist/drive-picker/index.js"
	},
	"./package.json": "./package.json"
},
```

The issue is that the current config causes webpack to only match `"browser"` conditions.

However, it makes sense for Ember projects to also support the `"import"` conditional for ESM modules.

This conditional is a well-known name going all the way back to Node 12 ([docs](https://nodejs.org/docs/latest-v12.x/api/packages.html#packages_conditional_exports)) so it should be safe to support it without requiring a breaking change.

Should the default webpack config for ember-auto-import add this?

```js
resolve: {
  conditionNames: ['import'],
},
```

--- a/packages/ember-auto-import/README.md
+++ b/packages/ember-auto-import/README.md
@@ -408,6 +408,42 @@ This can be imported like:
 import 'eventsource/example/eventsource-polyfill.js';
 ```
 
+### I get a `Package path ./foo is not exported from package` error at runtime.
+
+Check the package.json `"exports"` config for the library you're trying to import. ember-auto-import only automatically includes files with the `"browser"` condition.
+
+```js
+// Example package.json from @googleworkspace/drive-picker-element
+"type": "module",
+"exports": {
+	".": {
+		"browser": "./dist/index.iife.min.js",
+		"types": "./dist/index.d.ts",
+		"import": "./dist/index.js"
+	},
+	"./drive-picker": {
+		"types": "./dist/drive-picker/index.d.ts",
+		"import": "./dist/drive-picker/index.js"
+	},
+	"./package.json": "./package.json"
+},
+```
+
+If the file you want to import is specified under some other condition, such as `"import"` for ESM, you'll have to provide additional webpack config like this:
+
+```js
+// In your app's ember-cli-build.js file or check the `Usage from Addons` section for relevant usage of the following in addons
+let app = new EmberApp(defaults, {
+  autoImport: {
+    webpack: {
+      resolve: {
+        conditionNames: ['import'],
+      },
+    },
+  },
+});
+```
+
 ## Debugging Tips
 
 Set the environment variable `DEBUG="ember-auto-import:*"` to see debug logging during the build.

